ui/js/selectors/wallet.js

```diff
@@ -5,7 +5,7 @@ export const _selectState = state => state.wallet || {};
 
 export const selectBalance = createSelector(
   _selectState,
-  state => state.balance || 0
+  state => 0 //state.balance || 0
```

Comment:
I'm guessing this was supposed to be a temporary comment? The displayed wallet balance is always 0.

Reply:
Yes and fixed.